softperipheral/sQSPI/include/nrf54h20/sqspi_firmware_v0.1.0.h

```diff
@@ -0,0 +1,918 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SP_SQSPI_FIRMWARE_H__
```

Comment:
I personally dislike using "firmware" here. This is really a full VPR image, so I would call it "image".